airflow/api_connexion/endpoints/task_instance_endpoint.py

```diff
@@ -248,7 +248,7 @@ def post_clear_task_instances(dag_id: str, session=None):
         error_message = f"Dag id {dag_id} not found"
         raise NotFound(error_message)
     reset_dag_runs = data.pop('reset_dag_runs')
-    task_instances = dag.clear(get_tis=True, **data)
```

Comment:
I believe `tis` stands for `task instances`, so maybe `get_task_instances` to make it more explicit? 

Another problem, `ti` is also a "public" variable as it's used in xcom, task context, macros etc. Though in general, I prefer explicit over abbreviation, I don't think it's good change to include within this PR.

The same applies to the rest of `ti_instances`